README.md

```diff
@@ -57,9 +59,17 @@ You should be able to run `bash-handbook` at the command line now. This will ope
 
 If you are a developer, then you know the value of time. Optimizing your work process is one of the most important aspects of the job.
 
-In that path towards efficiency and productivity, we are often posed with actions that must be repeated over and over again, like taking a screenshot and uploading it to a server, processing text that may come in many shapes and forms, converting files between different formats, parsing a program's output and the list goes on. Enter bash, our savior.
+In that path towards efficiency and productivity, we are often posed with actions that must be repeated over and over again, like
+
+* taking a screenshot and uploading it to a server
+* processing text that may come in many shapes and forms
+* converting files between different formats
+* parsing a program's output
+* the list goes on ...
 
-**Bash** is a Unix shell written by [Brian Fox](https://en.wikipedia.org/wiki/Brian_Fox_(computer_programmer)) for the GNU Project as a free software replacement for the [Bourne shell](https://en.wikipedia.org/wiki/Bourne_shell). It was released in 1989 and has been distributed as the Linux and OS X default shell for a long time.
+Enter **Bash**, our savior.
+
+Bash is a Unix shell written by [Brian Fox](https://en.wikipedia.org/wiki/Brian_Fox_(computer_programmer) for the GNU Project as a free software replacement for the [Bourne shell](https://en.wikipedia.org/wiki/Bourne_shell). It was released in 1989 and has been distributed as the Linux and OS X default shell for a long time.
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3459374/12695098/cacb9fde-c74b-11e5-8652-3fd06bd331bd.png)

Fix this, please. The most of people read this book on GH, so it must looks perfect here.